udf_js/unzip.sql

```diff
@@ -0,0 +1,53 @@
+/*
+
+Unzips a GZIP string.   
+
+This implementation relies on the zlib.js library (https://github.com/imaya/zlib.js) and 
+the atob function for decoding base64.
+
+*/
+
+
+CREATE TEMP FUNCTION
+  udf_js_unzip (input BYTES)
```

Comment:
we should probably call this `gunzip` like the command line utility, just in case we have other compression types in the future:
```suggestion
  udf_js_gunzip (input BYTES)
```